modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -136,24 +139,51 @@ export default {
       });
       this.next = next;
       this.fieldState = fieldState;
+
+      this.$nextTick(() => {
```

Comment:
seems like a lot of careful steppin' to get the timing of this stuff right .. is there some simpler comparison?

Reply:
Simpler comparison where?

Reply:
This is in service of not emitting any document updates if the schema is still populating. Before it was emitting `input` during the schema population.

Reply:
As I recall, the `$nextTick` is used so that any changes in the editor (the parent component) have completed before we let this start emitting `input` events.